packages/worker/src/worker.ts

```diff
@@ -1245,8 +1251,7 @@ export class Worker {
                     // Fatal error means we cannot call into this workflow again unfortunately
                     if (!isFatalError) {
                       const externalCalls = await state.workflow.getAndResetSinkCalls();
-                      // TODO: state.info.searchAttributes are not updated here
```

Comment:
This once only applied to search attributes (hence the TODO) but we've added more mutable properties to info since.
There should be a better way to ensure that `info` doesn't go stale.
Maybe we should get it from workflow context when extracting the sink calls, that'll give us search attributes too.

Reply:
Yeah, that's what I had in mind. My initial take at this was to add the `workflowInfo` object from the sandbox context into `SinkCall` objects.

Reverted that however, as this is not only about sink calls. We're also exposing the worker side `workflowInfo` object on log messages, and eventually, we would make it available to payload encoders.

I'm now considering sending `workflowInfo` on activation completion instead.